seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -473,14 +473,21 @@ arbitrum,camelot,3,2023-10-15,0x810b6259d2171ca029a3ddb86dfbf682a930644342408219
+optimism,openocean,2,2023-06-30,0x4a823ca5c73264723a1326c9d3dedd0f1b2bbbc446e7f2c71ac5c998234b8ca2,9,0x1db2466d9f5e10d7090e7152b68d62703a2245f0,14.714535262567255,0x7f5c764cbc14f9669b88837ca1490cca17c31607,2
+optimism,openocean,2,2022-08-03,0x13875f629d4c608d034411333bf0f0095fb6ac41c219163ed8d359805a63fa05,25,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,11.319842,0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9,11.265711412351504
 celo,curve,1,2023-10-17,0x4cef035b424859b5d61b1da3c2f03febd83f03083c764fa1af860bf4e26d28b3,16,0x37f750b7cc259a2f741af45294f6a16572cf5cad,118810.850166,0x765de816845861e75a25fca122bb6898b8b1282a,118999.79999999999
 celo,curve,1,2023-10-15,0x0a524624a533492b91c1204004589225f697d37bf63af60fde884b226e8b1f26,14,0x37f750b7cc259a2f741af45294f6a16572cf5cad,10.329967,0x765de816845861e75a25fca122bb6898b8b1282a,10.34099704814135
 celo,curve,1,2023-10-16,0xba430c475c9ee7e8bd17c760f6a7833b0779781092033de73ca1bf3f7e86e7f8,43,0x37f750b7cc259a2f741af45294f6a16572cf5cad,5.189966,0x765de816845861e75a25fca122bb6898b8b1282a,5.197918504908161
 celo,curve,1,2023-10-15,0x28500eb8fe60219ee750bbea028e4ace3259aae1f350727c97311be3ac579dd4,13,0x37f750b7cc259a2f741af45294f6a16572cf5cad,432.240793,0x765de816845861e75a25fca122bb6898b8b1282a,432.6936029057536
-optimism,openocean,2,2023-09-04,0x1c2bc7857ce9acd8d6b018eea4b2e2affbda61e56916161b4348ef0603ff7935,23,0x350a791bfc2c21f9ed5d10980dad2e2638ffa7f6,82.11154686585886,0x7f5c764cbc14f9669b88837ca1490cca17c31607,500.376315
-optimism,openocean,2,2023-06-30,0x4a823ca5c73264723a1326c9d3dedd0f1b2bbbc446e7f2c71ac5c998234b8ca2,9,0x1db2466d9f5e10d7090e7152b68d62703a2245f0,14.714535262567255,0x7f5c764cbc14f9669b88837ca1490cca17c31607,2
-optimism,openocean,2,2022-08-03,0x13875f629d4c608d034411333bf0f0095fb6ac41c219163ed8d359805a63fa05,25,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,11.319842,0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9,11.265711412351504
+optimism,openocean,2,2023-05-30,0xc97e676905d6d2763fa55ac9aac6d6d85f6597901cf34711d6b65981b0cb0f75,7,0x7f5c764cbc14f9669b88837ca1490cca17c31607,3.193448,0x4200000000000000000000000000000000000006,0.00167453389173889
+optimism,openocean,2,2023-11-07,0xde2d7e6f02cfce8eb8a1bb752ce956cfcf81b7bf8380490bcdfeab9694b4e5ad,15,0x8700daec35af8ff88c16bdf0418774cb3d7599b4,2.9738578135579843,0x4200000000000000000000000000000000000006,0.003958725899153863
+optimism,openocean,2,2023-07-31,0x92188637370c0bf3595ae4bef248db90f43167483320fc9b21adb016cfb84e76,81,0x8b21e9b7daf2c4325bf3d18c1beb79a347fe902a,1438.894027919516,0x4200000000000000000000000000000000000042,14.86875
 base,maverick,1,2023-10-24,0xf76389f7730beb6a51a3e1881b0a9139fe8ec07d8403061fd52d5aaf1357ad00,12,0x50c5725949a6f0c72e6c4a641f24049a917db0cb,2955.014794642196128309,0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca,2954.059434
+zksync,maverick,1,2023-10-23,0xbe2700aac1174728513863a4f42d37aa9ac7a7d73389fa00a5abcfbd3570d5a6,4,0x2039bb4116B4EFc145Ec4f0e2eA75012D6C0f181,,0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4,
```

Comment:
By the way @ARDev097, why do you add this `zksync,maverick,1` seed record here？

Reply:
Sorry @Hosuke for it, but there is some conflict issue, so when i resolved it was added by mistakenly